### PR TITLE
Added ASCII entry

### PR DESCRIPTION
**Pull request by jcruz-io**
Just adding a little entry about ASCII meaning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ There are sometimes multiple words for the same thing, again - just to be confus
 
 
 **API** - Stands for Application Programming Interface, which provides a comprehensive layer of communication and callbacks to the original software layer, so that it becomes easier to integrate. Example- The google maps API allows you to integrate their maps to your apps through their API.
+
+**ASCII** - Stands for 'American Standart Code for Information Interchage'. It is the set of common characters use in electronic communication, based in English alphabet, numbers, punctuation symbols and non-printing control characters (like Escape, Tabulation, Backspace, End of Line, etc). This standard is the base line to newer sets of characters like UTF-8 standard, that extends ASCII including new characters like Latin or German characters by example.
```
Comment (FSipiano) [inline README.md]: Is this a typo - standart?

**Review by FSipiano — CHANGES_REQUESTED** [review]
Please fix the typo and I can merge

File: README.md (hunk 1 above)
Comment (jcruz-io) [reply README.md]: Sorry, my bad

Comment (FSipiano) [reply README.md]: No worries, thanks for fixing!

**Review by FSipiano — APPROVED** [review]
Perfect, thanks. Please create a pull request and I can merge

**Pull request merged**